application-services/custom/influxdb-export/README.md

```diff
@@ -0,0 +1,16 @@
+# EdgeX Foundry application service InfluxDB
+
+EdgeX Foundry application service to export readings to InfluxDB via MQTT.
+
+The pipeline simply consists of two functions:
+- conversion of EdgeX Event/Reading objects to Influx [line protocol](https://docs.influxdata.com/influxdb/v2.0/reference/syntax/line-protocol/)
+- export of results to MQTT Sender, which is a MQTT topic that Telegraf / Influx is positioned to listen to and ingest data from
+
+In this example, the MQTT Broker is assumed to exist on a different host and is accessed via port 1884.  The target topic is edgex/EdgeXEvents.  Modify the configuration.toml file to point the MQTT Sender to an alternate broker or topic.
```

Comment:
Why 1884 rather than the standard MQTT port of 1883?

Reply:
I had another broker at that port.  I changed it for consistency.